src/plugins/pcr/src/ExtractProductTask.cpp

```diff
@@ -240,7 +241,11 @@ void ExtractProductTask::run() {
     AnnotationTableObject *annotations = new AnnotationTableObject(productSequence.getName() + " features", dbiRef);
     annotations->addAnnotations(QList<SharedAnnotationData>() << getPrimerAnnotation(product.forwardPrimerMatchLength, U2Strand::Direct, productSequence.length()));
     annotations->addAnnotations(QList<SharedAnnotationData>() << getPrimerAnnotation(product.reversePrimerMatchLength, U2Strand::Complementary, productSequence.length()));
-    annotations->addObjectRelation(GObjectRelation(GObjectReference(sequenceObject), ObjectRole_Sequence));
+    GObjectReference ref(sequenceObject);
```

Comment:
Why adding a random non-existing URL  here helps to solve the problem?

Reply:
Impossible add object relation with empty url in GObjectReference. 
So we add temporary url for object to pass this.
As i understand this situation is only possible when this task started through WD.

Reply:
Actually, it looks like a crutch, but I don't have a better offer

Reply:
A long story short: hacks like this won't go.

If GObjectReference requires an URL the URL must be real. Why can't we re-use the URL of the document for the sequence object?